src/lib/components/radio-button/index.tsx

```diff
@@ -0,0 +1,70 @@
+import { useTheme } from '@emotion/react';
+import stylin from '@stylin.js/react';
+import React, { FC, PropsWithChildren, useState } from 'react';
+
+import { Box } from '../../elements';
+import { RadioCircleSVG } from '../../icons';
+import { Theme } from '../../theme';
+import {
+  RadioButtonElementProps,
+  RadioButtonProps,
+} from './radio-button.types';
+
+export const RadioButton: FC<PropsWithChildren<RadioButtonProps>> = ({
+  ...props
+}) => {
+  const { dark } = useTheme() as Theme;
+  const [selector, setSelector] = useState(props.checked || false);
+  const RadioButtonElement = stylin<RadioButtonElementProps>('input')();
+  return (
+    <Box display="flex" gap="1rem" flexWrap="wrap">
+      <RadioButtonElement
+        {...props}
+        type="radio"
+        display="none"
+        checked={selector}
+      />
+      <Box
+        width="2.5rem"
+        height="2.5rem"
+        color={
+          props.disabled
+            ? 'disabled'
+            : selector
+            ? 'accent'
+            : dark
```

Comment:
Here u must use same disabled version but with some opacity, just check the text field

Reply:
check again Sir